docs/proposals/20201020-capi-provider-operator.md

```diff
@@ -74,6 +74,9 @@ This proposal provides a solution that leverages a declarative API and an
 operator to empower admins to handle the lifecycle of providers within the
 management cluster.
 
+The operator will be developed in a separate repository and will have it's own release cycle. 
+The possibility of moving it into core repository as a sub-project will be evaluated later.
```

Comment:
```suggestion
```

I'd remove this for now

Reply:
We can, although it doesn't really add more information to the users that's why I was suggesting to remove it